### DISCUSSION TITLE
opentelemetry.io/v1alpha1 giving error

### DISCUSSION THREAD

**Discussion by Ripurwar80** [open-telemetry/opentelemetry-collector#6278] [Q&A]
I'm using opentelemetry.io/v1alpha1 api version in my collector how do I pull private images and attach service account. imagePullsecret and serviceAccountName are not supported by this api version.

**Comment by jpkrohling** [comment]
This belongs to the open-telemetry/opentelemetry-operator repository. Looks like I can't move this issue there.

cc @open-telemetry/operator-maintainers

**Comment by pavolloffay** [comment]
The service account can be set in the CR https://github.com/open-telemetry/opentelemetry-operator/blob/main/apis/v1alpha1/opentelemetrycollector_types.go#L89 which allows configuring image pull secret.
